Dockerfile.base

```diff
@@ -0,0 +1,38 @@
+FROM python:3.6-alpine
+LABEL maintainer "EGA System Developers"
+
+RUN apk add --no-cache \
+    libressl libressl-dev make gcc musl-dev libffi-dev postgresql-libs postgresql-dev git
+
+#################################################
+##
+## Install LocalEGA stuff
+##
+#################################################
+
+ARG LEGA_GID=1000
+RUN addgroup -g ${LEGA_GID} lega && \
+    adduser -D -G lega lega
+
+# RUN git clone https://github.com/EGA-archive/crypt4gh.git /root/crypt4gh && \
+#     pip3.6 install -r /root/crypt4gh/requirements.txt && \
+#     rm -rf /root/crypt4gh
+
+COPY setup.py /root/LocalEGA/setup.py
+COPY lega /root/LocalEGA/lega
+COPY requirements.txt /root/LocalEGA/requirements.txt
+
+RUN pip install -r /root/LocalEGA/requirements.txt && \
```

Comment:
I still classifies these images as `development grade`, not production ready.
This is in essence a monolithic application packaged as a container.

Reply:
So?

It's not what this PR claims

Reply:
Just a comment, but still something we need to fix down the road.